gamedata/descriptions/levels/naughtcoin.md

```diff
@@ -0,0 +1,6 @@
+NaughtCoin is a play on words. It's an ERC20 compliant token for ether-_naut_ and the worth of these tokens after you hack them, _naught_. You hold all of the tokens, but you'll only be able to transfer them after a 10 year lockout period. Can you figure out how to get your hands on them before the lockout period finishes?
```

Comment:
The description of the level is a bit confusing.

1) The objective of the level should be more clearly stated. You say "Can you figure out how to get your hands ON them before the lockout period finisies?" Shouldn't it be OFF instead of on. I would more clearly state that the objective is to hold 0 tokens to get to the next level.
2) I don't quite get what the play on words is. Perhaps you could make it a bit clearer. Is it because you are supposed to hold naught coins to solve the level?